pandas/core/generic.py

```diff
@@ -9961,14 +9990,75 @@ def _check_percentile(self, q):
               2016      2015      2014
         GOOG   NaN -0.151997 -0.086016
         APPL   NaN  0.337604  0.012002
+
+        In a DataFrame with NAs, when computing the percentage change with
```

Comment:
Can you add a `skipna=False` example to highlight difference?

Reply:
@WillAyd are you sure that example is useful?

```
In [6]: df.pct_change(skipna=False)                                                                                                            
Out[6]: 
          a   b   c
0       NaN NaN NaN
1  0.011111 NaN NaN
2       NaN NaN NaN
3       NaN NaN NaN
4       NaN NaN NaN
5       NaN NaN NaN
```